tests/Behat/Resources/suites.yml

```diff
@@ -134,26 +154,11 @@ default:
                 - sylius.behat.context.hook.doctrine_orm
 
                 - sylius.behat.context.transform.shared_storage
-                - webgriffe_sylius_akeneo.behat.context.transform.queue_item
 
                 - sylius.behat.context.setup.admin_security
                 - webgriffe_sylius_akeneo.behat.context.setup.queue
 
-                - webgriffe_sylius_akeneo.behat.context.ui.admin.managing_queue_items
                 - sylius.behat.context.ui.admin.notification
 
             filters:
                 tags: "@managing_queue_items && @ui"
```

Comment:
It doesn't let me suggested this change: the entire "ui_managing_queue_items" node should be removed